src/pages/components/video/video.card.js

```diff
@@ -54,23 +64,23 @@ const Video = (
     >
       <div className="card-media" onClick={() => onVideoClick(video)}>
         <img src="https://picsum.photos/200/250/" alt="video" />
```

Comment:
Suggestion: can add lazy loading just to prevent blocking of DOM in case the image is very large

Reply:
no, otherwise it looks broken card while getting the image